src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.ts

```diff
@@ -379,22 +407,43 @@ export class ExampleModelingSubmissionComponent implements OnInit {
 
         this.tutorParticipationService.assessExampleSubmission(exampleSubmission, this.exerciseId).subscribe(
             () => {
+                this.markAllFeedbackToCorrect();
                 this.jhiAlertService.success('artemisApp.exampleSubmission.assessScore.success');
             },
             (error: HttpErrorResponse) => {
                 const errorType = error.headers.get('x-artemisapp-error');
 
-                if (errorType === 'error.tooLow') {
-                    this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooLow');
-                } else if (errorType === 'error.tooHigh') {
-                    this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooHigh');
+                if (errorType === 'error.invalid_assessment') {
+                    this.markAllFeedbackToCorrect();
+
+                    // Mark all wrongly made feedbacks accordingly.
+                    const correctionErrors: FeedbackCorrectionError[] = JSON.parse(error['error']['title'])['errors'];
+                    correctionErrors.forEach((res) => {
+                        const validatedFeedback = this.feedbacks.find((feedback) => feedback.reference === res.reference);
+                        if (validatedFeedback != undefined) {
+                            validatedFeedback.correctionStatus = res.type;
+                        }
+                    });
+
+                    this.assessmentEditor.resultFeedbacks = this.feedbacks;
+
+                    const msg =
```

Comment:
`errorMessage`  might be better instead of acronym

Reply:
Good catch. I changed it 👍